plugins/inputs/systemd_units/subcommand_list.go

```diff
@@ -0,0 +1,88 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package systemd_units
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+)
+
+// Gather parses systemctl outputs and adds counters to the Accumulator
+func parseListUnits(_ *SystemdUnits, buffer *bytes.Buffer, acc telegraf.Accumulator) error {
```

Comment:
Please make the accumulator the first parameter. This is an unwritten rule (context, accumulator, remaining) and allows to spot the usage of the accumulator easily.